beads_model/src/main/java/com/beads/model/dao/ProductCommentDaoImpl.java

```diff
@@ -0,0 +1,85 @@
+package com.beads.model.dao;
+
+import com.beads.model.constant.CommentStatus;
+import com.beads.model.domain.Product;
+import com.beads.model.domain.ProductComment;
+import java.util.List;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ProductCommentDaoImpl extends BaseDao implements ProductCommentDao {
+
+  /**
+   * This method save comment for product,
+   * if comment with same parameter exist it will be updated.
+   *
+   * @param productComment which must be save
+   *
+   * @return id of saved comment
+   */
+  @Override
+  public int saveOrUpdateComment(ProductComment productComment) {
+    getSession().saveOrUpdate(productComment);
+    return productComment.getId();
+  }
+
+  /**
+   * This method load comment by commentId.
+   *
+   * @param id which need load
+   *
+   * @return loaded comment
+   */
+  @Override
+  public ProductComment loadCommentById(Integer id) {
```

Comment:
better rename it to `getCommentById`
if you use hibernate `get` because hibernate has `load` method